assignments/objects.js

```diff
@@ -19,27 +19,79 @@ const example = {
 
 // Write your intern objects here:
 
+const mitzi = {
```

Comment:
Do you think that you can write these objects in a way that they are nested into one?